src/api/go/pxapi/client.go

```diff
@@ -79,12 +81,34 @@ func NewClient(ctx context.Context, opts ...ClientOption) (*Client, error) {
 		opt(c)
 	}
 
+	if (strings.Contains(c.directAddr, "0.0.0.0") == true) {
+		if err := c.initDirectClient(ctx); err != nil {
+			return nil, err
+		}
+		return c, nil
+	}
+
 	if err := c.init(ctx); err != nil {
 		return nil, err
 	}
 	return c, nil
 }
 
+
+// initDirectClient is for establishing gRPC connection to standalonePEM
+func (c *Client) initDirectClient(ctx context.Context) error {
+        conn, err := grpc.Dial(c.directAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
I think we should take the credentials in as another `ClientOption` eg `WithDirectCreds`